### PR TITLE
Custom ktlint rule no print

### PR DESCRIPTION
**Pull request by jaahon**
#### Summary

Add a custom ktlint rule that complains about `print` and `println` calls in the code.

### CODE REVIEW

File: service/custom-ktlint-rules/settings.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// SPDX-FileCopyrightText: 2017-2020 City of Espoo
```
Comment (mikkopiu) [inline service/custom-ktlint-rules/settings.gradle.kts:1]: Wrong year:
```suggestion
// SPDX-FileCopyrightText: 2017-2021 City of Espoo
```

File: service/custom-ktlint-rules/src/main/kotlin/fi/espoo/evaka/ktlint/NoPrint.kt [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: 2017-2020 City of Espoo
```
Comment (mikkopiu) [inline service/custom-ktlint-rules/src/main/kotlin/fi/espoo/evaka/ktlint/NoPrint.kt:1]:
```suggestion
// SPDX-FileCopyrightText: 2017-2021 City of Espoo
```

File: service/custom-ktlint-rules/src/test/kotlin/fi/espoo/evaka/ktlint/NoPrintTest.kt [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package fi.espoo.evaka.ktlint
```
Comment (mikkopiu) [inline service/custom-ktlint-rules/src/test/kotlin/fi/espoo/evaka/ktlint/NoPrintTest.kt:1]: Missing license header, add with `./bin/add-license-headers.sh`

File: service/custom-ktlint-rules/src/main/kotlin/fi/espoo/evaka/ktlint/NoPrint.kt [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+            canBeAutoCorrected: Boolean
+        ) -> Unit
+    ) {
+        if (node.elementType == CALL_EXPRESSION) {
+            val callExpression = node.psi as KtCallExpression
+            val isPrintCall = callExpression.children.any { e ->
+                e is KtReferenceExpression && printFunctions.contains(e.text)
+            }
+            if (isPrintCall) {
+                emit(node.startOffset, "Do not use print or println for logging", false)
```
Comment (mikkopiu) [inline service/custom-ktlint-rules/src/main/kotlin/fi/espoo/evaka/ktlint/NoPrint.kt:31]: Maybe we should direct people to the correct method (i.e. using `mu.KotlinLogging.logger`)

**Review by mikkopiu — CHANGES_REQUESTED** [review]
Mostly looking good, some comments

File: service/custom-ktlint-rules/src/main/kotlin/fi/espoo/evaka/ktlint/NoPrint.kt (hunk 4 above)
Comment (jaahon) [reply service/custom-ktlint-rules/src/main/kotlin/fi/espoo/evaka/ktlint/NoPrint.kt:31]: Sure, good idea

**Review by mikkopiu — APPROVED** [review]
LGTM

**Pull request merged**